### PR TITLE
[ui] (Fabric) Updating guidance about Fabric UI control selection

### LINKED ISSUE #2283: Documentation on Fabric components recommended for use in Office add-ins seems inconsistent w.r.t. MessageBar, Callout and CommandBar
The [section titled “Use Fabric Components”](https://docs.microsoft.com/en-us/office/dev/add-ins/design/office-ui-fabric#use-fabric-components) in the documentation (quoted below) is a bit confusing: it first gives examples of the kinds of components Fabric provides that can be used to build an add-in, and then gives a bullet list of the components that are recommended for use in add-ins; the first five example components are also included in the list (`Button`, `Checkbox`, … , `Breadcrumb`), but the last two are not (`MessageBar` and `Callout`) so it’s somewhat unclear whether these two are or are not actually recommended. The `CommandBar` component is also missing from the list, while the page on [“Task panes”](https://docs.microsoft.com/en-us/office/dev/add-ins/design/task-pane-add-ins#best-practices) has a “best practice” that states: “Include some navigational or commanding element such as the CommandBar or Pivot at the top of your add-in.”

> **Use Fabric Components**
>
> Fabric provides a variety of UX components that you can use to build your add-in, including the following types of components:
> 
> * Input components - for example, Button, Checkbox, and Toggle
> * Navigation components - for example, Pivot and Breadcrumb
> * Notification components - for example, MessageBar and Callout
> 
> Not all Fabric components are recommended for use in add-ins. Here is a list of Fabric React UX components that we recommend for use in an add-in:
> 
> * Breadcrumb
> * Button
> * Checkbox
> * ChoiceGroup
> * Dropdown
> * Label
> * List
> * Pivot
> * TextField
> * Toggle

### PR DESCRIPTION
**Pull request by AlexJerabek**
Fixes #2283 

This PR corrects the line recommending Fabric components your an add-in. It, instead, specifies that the given list is a general recommendation, instead of a more exclusionary list.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[329646a](https://github.com/OfficeDev/office-js-docs-pr/commits/329646ae60aff3dc66a5d4f88bdaab31bf6d1790)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/office-ui-fabric.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Fabric/docs/design/office-ui-fabric.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/office-ui-fabric?branch=pr-en-us-2294) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010292250373203-2294%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qFSZlb2FWqtNZiZDq2zGB7OyGmi8UHxzDqqKRqQCe2s%3D&st=2020-10-29T22%3A46%3A15Z&se=2020-11-29T22%3A51%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/design/office-ui-fabric.md [hunk 1]
```diff
@@ -1,52 +1,48 @@
 ---
 title: Office UI Fabric in Office Add-ins 
 description: 'Get an overview of how to use the Office UI Fabric components in Office Add-ins.'
-ms.date: 04/20/2020
+ms.date: 10/20/2020
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/office-ui-fabric.md:4]:
```suggestion
ms.date: 10/29/2020
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Adding @Rick-Kirkham as a reviewer too (I'll forward you an internal thread with additional questions).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[efb0338](https://github.com/OfficeDev/office-js-docs-pr/commits/efb0338d529dc582f8f7fd055c2ba9f422236340)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/office-ui-fabric.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Fabric/docs/design/office-ui-fabric.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/office-ui-fabric?branch=pr-en-us-2294) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011052008332202-2294%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=E0r6GQkaxXcAgFBM%2BMJSLxpvw4%2BxhJ4WYaaOdTUfv6s%3D&st=2020-11-05T20%3A04%3A12Z&se=2020-12-06T20%3A09%3A12Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by Rick-Kirkham — APPROVED** [review]
looks good

**Pull request merged**